### ISSUE TITLE
Support for UV (included in Aeotec/Aeon Labs Multisensor 6)

### ISSUE THREAD

**Issue by sopelt** [genielabs/zwave-lib-dotnet#25]
When the Aeotec Multisensor 6 submits its UV value, the event is not processed by ZWaveLib.
Its code is 27 and seems to be missing/unhandled in [SensorValue.cs](https://github.com/genielabs/zwave-lib-dotnet/blob/5b7c2b4b1e9fa21b7ada1a1a002bff24728b5710/ZWaveLib/Values/SensorValue.cs#L28).

Most of all I would like to get rid of the message but actually fully supporting this (quite common) device would be nice.

Anyone keen on looking into that or should I go ahead and have a look and create a PR?

Are there any plans for new releases? The NuGet package is quite stale compared to the repo.

Thank you.

**Comment by Bounz** [comment]
Received this device two days ago)
I'll implement UV values.

About new release: are you talking only about the release of ZWaveLib, or you are also interested in MIG and HG?

**Comment by genemars** [comment]
@Bounz @sopelt , I'll wait for this update to both release zwavelib nuget package and new HG.

**Comment by Bounz** [comment]
Here you are https://github.com/genielabs/zwave-lib-dotnet/pull/26.

**Comment by sopelt** [comment]
@Bounz , thank you for the swift reply and coding work!
I am currently using a kind of homebrew solution for monitoring and recording temps and humidity based on ciseco/wirelessthings XRF ... given that they went out of business I had to extend using something else and that got me into z-wave. That is how I ended up using ZWaveLib but I guess I should look at MIG + HG.

Thank you again ... much appreciated.

@genemars perfect! Thank you. I think somewhat regular updates on NuGet are great to communicate the health of the project.
